src/Command/I18nExtractCommand.php

```diff
@@ -857,7 +857,7 @@ protected function _searchFiles(): void
             }
             $this->_files = array_merge($this->_files, $files);
         }
-        $this->_files = array_unique($this->_files);
+        $this->_files = array_values(array_unique($this->_files));
```

Comment:
There were a lot of these cases, where either the property or the return type of the method requires a `list` instead of an `array`. Therefore my quickest way to achieve this was to wrap all these calls in `array_values`.

Not sure if/how much performance this costs if the array behind it is already a list.

Reply:
I would not think it is wise to add this.
Then we should rather not use list<> and use normal array<> here?
Imo tooling seems a bit too pedantic on the key here as we only care about the value im these cases.

Reply:
well its about knowing if its a simple array list, or an associative array. Technically one could provide either one for certain method arguments since php native everything is just an `array`.

therefore phpstan just reports it (which it does because we have such a high phpstan level active).

I do like narrowing it if its possible, we maybe just have to find a way to tell phpstan, that its actually a `list` in the first place instead of having to force it by `array_values` or other array methods.

Reply:
List<> is about non assoc list whereas we dont care actually. We just dont call it by key thus not assoc by Design even if it has mixed keys.
So essentially we are missing a values<> or alike. Until then it seems better to keep using array<> instead of the (wrong) list.

Reply:
This also affects lots of user land code.
So I for once am sure not in favor of having to add totally unnecessary function calls.

Reply:
We should still think about reverting list usage here too.
As I said: extending plugin and user land code would also have tons of baseline issues otherwise.

Reply:
Well they also have a lot of related Bugs still
https://github.com/phpstan/phpstan/issues/11917
So maybe some it solved itself until then

Reply:
Yeah, reverting `list<foo>` to `array<foo>` might be a good idea. There's rarely a use case where having the keys as contiguous numeric indexes is necessary.

Reply:
We have to do it for the ones getting alerts now.
We can revisit the remaining ones later in 5.next until release.